sormas-api/src/main/java/de/symeda/sormas/api/immunization/ImmunizationIndexDto.java

```diff
@@ -211,12 +208,4 @@ public Date getRecoveryDate() {
 	public void setRecoveryDate(Date recoveryDate) {
 		this.recoveryDate = recoveryDate;
 	}
-
-	public Disease getDisease() {
```

Comment:
The disease still has to be a part of the directory and actually needs to be added as a column (I didn't think about that when writing down that the "select a disease" hint needs to be removed)